src/components/IRSTableCell/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
This component is responsible for rendering the table cell that holds the jurisdiction name (and that does the drilling down).